dashboard/dashbord.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Dashboard</title>
+        <link rel="stylesheet" href="style.css">
+        <script type="text/javascript" src="../js/dashboard.js"></script>
+    </head>
+    <body>
+        <header class="hb-header">
+            <div class="header-wrapper-top">
+                <div class="header-image-part">
+                    <div class="header-top">
+                        <div class="logo" onclick="window.location.href='http:\/\/127.0.0.1:5501\/foodwebsite\/food-website.html'">
+                            <img src="../img/logo.png" alt="Coffee Logo" class="logo-icon"></img>
```

Comment:
this can be a self closing tag
`<img src="../img/logo.png" alt="Coffee Logo" class="logo-icon" \>`